source/blender/gpu/intern/gpu_material.c

```diff
@@ -1375,9 +1375,11 @@ static void do_material_tex(GPUShadeInput *shi)
 				mtex->ofs[1] + 0.5f - 0.5f * mtex->size[1],
 				0.0f
 			};
+			mtex->ofs[0] = ofs[0];
```

Comment:
I'm not 100% sure about that. Maybe it would be good to see if mtex->ofs is used elsewhere.